easybuild/easyconfigs/g/GraphicsMagick/GraphicsMagick-1.3.45-GCCcore-13.3.0.eb

```diff
@@ -15,11 +15,11 @@ source_urls = [
 ]
 sources = [SOURCE_TAR_XZ]
 patches = [
-    'GraphicsMagick_pkgconfig_libtiff.patch'
+    'GraphicsMagick_pkgconfig_libtiff-4.patch'
 ]
 checksums = [
     {'GraphicsMagick-1.3.45.tar.xz': 'dcea5167414f7c805557de2d7a47a9b3147bcbf617b91f5f0f4afe5e6543026b'},
-    {'GraphicsMagick_pkgconfig_libtiff.patch': '25b4c5361f30e23c809a078ac4b26e670d2b8341496323480037e2095d969294'},
+    {'GraphicsMagick_pkgconfig_libtiff-4.patch': 'fddd53421387b410377d3147845cb7774b8c2cd1769865568c304237f85b2157'},
```

Comment:
pls add the GraphicsMagick version in the name of the patch?

Reply:
done